RosettaJdbi3/src/test/java/com/hubspot/rosetta/jdbi3/TestDao.java

```diff
@@ -13,6 +13,24 @@ public interface TestDao extends SqlObject {
   @SqlQuery("SELECT * FROM test_table")
   List<TestObject> getAll();
 
+  @SqlQuery("SELECT * FROM test_list_table")
+  List<TestListObject> getAllList();
+
+  @SqlQuery("SELECT * FROM test_list_table WHERE value IN (<values>)")
+  List<TestListObject> getWithValue(@BindListWithRosetta("values") List<TestEnum> values);
+
+  @SqlQuery("SELECT * FROM test_list_table WHERE value NOT IN (<values>)")
+  List<TestListObject> getWithoutValueEmptyToNull(@BindListWithRosetta(value = "values", onEmpty = EmptyHandling.NULL) List<TestEnum> values);
+
+  @SqlQuery("SELECT * FROM test_list_table WHERE value NOT IN (<values>)")
+  List<TestListObject> getWithoutValueEmptyToVoid(@BindListWithRosetta(value = "values", onEmpty = EmptyHandling.VOID) List<TestEnum> values);
+
+  @SqlQuery("SELECT * FROM test_list_table WHERE value IN (<values>)")
+  List<TestListObject> getWithFieldValue(@BindListWithRosetta(value = "values", field = "value") List<TestListObject> values);
+
   @SqlUpdate("INSERT INTO test_table (id, name) VALUES (:id, :name)")
   int insert(@BindWithRosetta TestObject object);
+
+  @SqlUpdate("INSERT INTO test_list_table (id, value) VALUES (:id, :value)")
+  int insert(@BindWithRosetta TestListObject object);
```

Comment:
This test isn't testing new functionality, right?

Reply:
Yea, it's just used to populate the table that we're querying against with the new annotation